lib/internal/errors.js

```diff
@@ -781,6 +791,40 @@ const fatalExceptionStackEnhancers = {
   }
 };
 
+// Ensures the printed error line is from user code.
+let _kArrowMessagePrivateSymbol, _setHiddenValue;
+function setArrowMessage(err, arrowMessage) {
+  if (!_kArrowMessagePrivateSymbol) {
+    ({
+      arrow_message_private_symbol: _kArrowMessagePrivateSymbol,
+      setHiddenValue: _setHiddenValue,
+    } = internalBinding('util'));
+  }
+  _setHiddenValue(err, _kArrowMessagePrivateSymbol, arrowMessage);
+}
+
+// Hide stack lines before the first user code line.
+function hideLeadingInternalFrames(error, stackFrames) {
+  let frames = stackFrames;
+  if (typeof stackFrames === 'object') {
```

Comment:
nit: `%Array.isArray%` would be more readable (or maybe `stackFrames !== undefined` would be enough?).
```suggestion
  if (ArrayIsArray(stackFrames)) {
```

Reply:
This came from a line of code in repl written by @devsnek. Wasn't sure if there was some original reason for it.